std_msgs/Header header
std_msgs/Header image_input_header
FaceClassification[] faces
